src/Flubit/Client/Client.php

```diff
@@ -432,11 +434,16 @@ public function createSingleProduct($productData)
     /**
      * {@inheritdoc}
      */
-    public function updateProducts($productData)
+    public function updateProducts($productData, $type = null)
     {
+        $type = $type ? $type : 'update';
+
         $request = $this->getPostRequest(
             sprintf('products/feed.%s',$this->responseFormat),
-            $productData
+            $productData,
+            [
+                'type' => $type
+            ]
         );
```

Comment:
The type defines the method name. updateProducts: type=update, createProducts: type=create. It should be a case of adding a replaceProducts method with type=replace or just having one method that can receive a type argument that could be create, update or replace.